.github/CONTRIBUTING.md

```diff
@@ -382,7 +382,7 @@ Once you sign a CLA, all your existing and future pull requests will have the st
 [testing-guidelines]: ../docs/testing-guidelines/testing-guidelines.md
```

Comment:
@bergmeister, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Common Tests - Validate Markdown Files.Should not have errors in any markdown files
```powershell
Expected strings to be the same, because /home/vsts/work/1/s/.github/SECURITY.md: 1: MD041/first-line-h1 First line in file should be a top level heading [Context: "If you believe that there is a..."]
 (See https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md for an explanation of the error codes), but they were different.
Expected length: 9
Actual length:   272
Strings differ at index 0.
Expected: '--EMPTY--'
But was:  '/home/vsts...'
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown.tests.ps1: line 97
97:         $combinedMarkdownErrors | Should -BeExactly "--EMPTY--" -Because $combinedMarkdownErrors
```

Reply:
@TravisEz13 @SteveL-MSFT `PoshChan` incorrectly marked this test failure, the test failure originated from `SECURITY.md` and not `CONTRIBUTING.md`